client/dashboard/components/notice/index.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Button, ExternalLink } from '@wordpress/components';
+import Notice from './index';
+
+const meta = {
+	title: 'client/dashboard/Notice',
+	component: Notice,
+	tags: [ 'autodocs' ],
+	argTypes: {
+		density: {
+			control: { type: 'radio' },
+			options: [ 'low', 'medium', 'high' ],
+		},
+		variant: {
+			control: { type: 'radio' },
+			options: [ 'info', 'warning', 'success', 'error' ],
+		},
```

Comment:
Are these options needed? I assumed Storybook would be able to auto-generate from the prop types